ui/components/app/asset-list/asset-list.js

```diff
@@ -120,48 +106,8 @@ const AssetList = ({ onClickAsset }) => {
   const balanceIsZero = Number(totalFiatBalance) === 0;
   const isBuyableChain = useSelector(getIsBuyableChain);
   const shouldShowBuy = isBuyableChain && balanceIsZero;
```

Comment:
@georgeweiler This `shouldShowBuy` variable is causing linting to fail on MMI builds https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/76477/workflows/4f9e8d7b-790e-47a4-9a7b-57cc1013528e/jobs/2639253

> scripts:core:dist:standardEntryPoints: Error: MetaMask build: Lint errors encountered for transformed file "/home/circleci/project/ui/components/app/asset-list/asset-list.js":

>    no-unused-vars
 >   'shouldShowBuy' is assigned a value but never used
 
 The declaration needs to be inside a `///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)` code fence